lib/table.ts

```diff
@@ -168,9 +175,9 @@ export default class Table {
       url: `/tables/${name}/orders`,
       method: "GET",
       headers: {
-        "doshii-location-id": locationId,
+        "doshii-location-id": locationId
       },
-      params: filters,
+      params: filters
```

Comment:
Unrelated, but while we have the opportunity, can you please remove this `params` from this endpoint? We don't support the `status` query parameter on this endpoint, so it's incorrectly defined here. 🤔

Reply:
Ignore this comment, I must have been looking at a different section of the code base. This is supported and should remain. 😕